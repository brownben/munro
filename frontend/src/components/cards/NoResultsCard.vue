<template>
  <section class="mt-2 text-center card">
    <div class="inline-block w-full">
      <img
        src="@/assets/images/UndrawLost.svg"
        class="object-contain w-full mx-auto max-h-64"
        alt="Person Lost looking at Map"
      />
    </div>

    <h1 class="mt-8 text-3xl font-bold text-gray-900 font-heading">
      Sorry, I Couldn't Find Any Results
    </h1>
  </section>
</template>
<style lang="postcss" scoped>
img {
  width: 100%;
  max-height: 14rem;
}
</style>
