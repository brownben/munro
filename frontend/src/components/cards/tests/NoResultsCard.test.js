import { mount } from '@vue/test-utils'
import NoResultsCard from '@/components/cards/NoResultsCard'

test('Is a Vue Instance', () => {
  const wrapper = mount(NoResultsCard)
  expect(wrapper.isVueInstance()).toBeTruthy()
})

test('Renders Correctly', () => {
  const wrapper = mount(NoResultsCard)
  expect(wrapper.element).toMatchSnapshot()
})
