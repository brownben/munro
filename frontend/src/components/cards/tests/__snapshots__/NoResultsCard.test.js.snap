// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Correctly 1`] = `
<section
  class="mt-2 text-center card"
>
  <div
    class="inline-block w-full"
  >
    <img
      alt="Person Lost looking at Map"
      class="object-contain w-full mx-auto max-h-64"
      src="@/assets/images/UndrawLost.svg"
    />
  </div>
   
  <h1
    class="mt-8 text-3xl font-bold text-gray-900 font-heading"
  >
    
    Sorry, I Couldn't Find Any Results
  
  </h1>
</section>
`;
